docs/reference/search/search-your-data/search-application-api.asciidoc

```diff
@@ -36,11 +36,116 @@ Learn more by reading about <<search-template,search templates>>.
 
 If no template is stored with a search application, a minimal <<search-application-api-default-template, default search template>> will be applied at search time.
 The default template implements a simple search use case.
-You can check your query parameters against the current template using the <<search-application-render-query,Render search application query>> API call.
+
+To create a search application with the default template, issue a <<put-search-application, create or update Search Application>> request without specifying a template:
+
+////
+
+[source,console]
+--------------------------------------------------
+PUT index1
+{
+  "mappings": {
+    "properties": {
+      "image-vector": {
+        "type": "dense_vector",
+        "dims": 3
+      }
+    }
+  }
+}
+
+PUT index2
+
+PUT _application/search_application/my_search_application
+{
+  "indices": [ "index1", "index2" ],
+  "template": {
+    "script": {
+      "source": {
+        "query": {
+          "query_string": {
+            "query": "{{query_string}}",
+            "default_field": "{{default_field}}"
+          }
+        }
+      },
+      "params": {
+        "query_string": "*",
+        "default_field": "*"
+      }
+    }
+  }
+}
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE _application/search_application/my_search_application
+
+DELETE index1
+
+DELETE index2
+--------------------------------------------------
+// TEARDOWN
+////
+
+[source,console]
+----
+PUT _application/search_application/my_search_application
+{
+  "indices": ["index1", "index2"]
+}
+----
+// TEST[warning:Using default search application template which is subject to change. We recommend storing a template to avoid breaking changes.]
+
+You can then use the <<get-search-application, get search application>> API call to view your newly created search application, which will also include the default template that was created for you:
+
+[source,console]
+----
+GET _application/search_application/my_search_application
+----
+// TEST[continued]
+// TEST[warning:Using default search application template which is subject to change. We recommend storing a template to avoid breaking changes.]
+
+In this case, the response would be:
+
+[source,console-result]
+----
+{
+  "name": "my_search_application",
+  "indices": [
+    "index1",
+    "index2"
+  ],
+  "updated_at_millis": 1715802354482,
+  "template": {
+    "script": {
+      "source": """{
```

Comment:
The error message shared in Slack seems to complain about `source` containing a string here rather than a map